package.json

```
@@ -28,16 +28,16 @@
         "live-server": "^1.2.1",
         "marked": "^2.1.3",
         "match-all": "^1.2.5",
-        "react-docgen": "^5.4.0",
+        "react-docgen": "^6.0.0-alpha.0",
```

Let's add `ast-types` as an explicit dependency:
```suggestion
        "react-docgen": "^6.0.0-alpha.0",
        "ast-types": "^0.14.2",
```

---

Woops! Thought I did 😅